### ISSUE TITLE
Symfony 7

### ISSUE THREAD

**Issue by SaItiplouf** [Setono/easyadmin-editorjs-bundle#3]
Hey, I would like to be able to use your project since CKEditor is not installable on symfony 7, but your bundle doesn't work too :/

**Comment by loevgaard** [comment]
Hi, @SaItiplouf 

I have opened a PR for fixing this issue. Right now the PR is blocked by another PR that I don't have any power over :)

**Comment by SaItiplouf** [comment]
And could you add any tool like : https://github.com/flaming-cl/editorjs-text-color-plugin
There is not that much options to edit text, color, size, font maybe? ect, I've tried modifying it myself but it's out of my capacity
;)

And is there a way to display it as Html in the Index Page of the CrudController ? Because we can only read JSON on the crud controller and we have to go into EDIT page to see the content.
